### ISSUE TITLE
Backpropogation Pass Question

### ISSUE THREAD

**Issue by MichaelGregg** [cisc3410/4-review#12]
@rivlev For the forward pass we got .17, but according to the training data, its supposed to be 1, so we do 1-0.17 and that gives us what? I am confused about the terms error, cost and loss in this context.  Specifically which ones we need to know how to calculate for use in the back pass.  Also, is there a resource that i can access that explains the process of the back pass for this question?

**Comment by irinafedorovacisc** [comment]
Having the same question "

**Comment by midwestmonika** [comment]
.17 is the output of the neural network on the first pass. 
Loss = 1 - 0.17 = 0.83
There are many ways to calculate the error, but we do (loss^2)/2
Error: (.83^2)/2 = 0.34

Error is used in the equations of the backpropagation when updating the weights in the neural network.

**Comment by midwestmonika** [comment]
Also, this is the video that we watched in class about backpropagation: https://www.youtube.com/watch?v=GlcnxUlrtek

**Comment by rivlev** [comment]
terms are actually the other way around
Error = 1-0.17 = 0.83
Loss/Cost = (.83^2)/2 = 0.34
Derivative of the Loss function = -Err = -0.83

Neural network solution now posted [here](https://photos.app.goo.gl/8RphLbBU8OtJ8iPJ2)
